geometry_msgs/Pose pose
---
gvg_explore/EdgeList edgelist